image-streams.yaml

```diff
@@ -0,0 +1,170 @@
+kind: List
+apiVersion: v1
+metadata:
+  name: image-streams
+  annotations:
+    description: ImageStream definition for Red Hat AMQ Streams 1.0
+    openshift.io/provider-display-name: Red Hat, Inc.
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: amqstreams10-clustercontroller-openshift
+    annotations:
+      openshift.io/display-name: AMQ Streams Cluster Controller     
+      openshift.io/provider-display-name: Red Hat, Inc.
+      version: 1.4.11
+  labels:
+    xpaas: 1.4.11
```

Comment:
Why xPaas and why 1.4.11?

Reply:
This label is for keeping our template and image-stream versions in sync with those of the other middleware(xPass) products ( https://github.com/jboss-openshift/application-templates ). Every Cloud Enablement(CE) sprint, this version is bumped. 

Although, since responsibilities of these images have changed from CE to the individual product teams, we might not need to follow this versioning scheme anymore. Hold on this PR until I figure this out.